spec/20-http_request_header_format.md

```diff
@@ -116,6 +120,8 @@ Vendors MUST ignore the `traceparent` when the `parent-id` is invalid (for examp
 
 #### trace-flags
 
+The current version of this specification (`00`) supports only two flags: `sampled` and `random-trace-id`.
+
```

Comment:
cannot make comment below, on line 135 we can change abstract example to a specific example with the random-trace-id flag. Saying that 0 and 2 represent not sampled, 1 and 3 - sampled